test/TestExpectations.json

```diff
@@ -2333,12 +2291,6 @@
     "parameters": ["cdp", "firefox"],
     "expectations": ["FAIL"]
   },
-  {
```

Comment:
is this handled by another expecation?

Reply:
Yeah, I have consolidate all the passing expectation into. That way we have one (or 2) thing affecting a test rather.
```json
  {
    "testIdPattern": "[navigation.spec] *",
    "platforms": ["darwin", "linux", "win32"],
    "parameters": ["webDriverBiDi"],
    "expectations": ["PASS"]
  }
  ```